src/components/Hero.vue

```diff
@@ -20,9 +20,7 @@ const { darkModeActive } = storeToRefs(state);
         <span class="extra-bold">studie</span>vereniging voor HBO-ICT van Hogeschool Utrecht
       </h1>
       <div class="subtitle-items">
-        <a class="button primary rounded" href="https://leden.conscribo.nl/svIndicium/aanmeldenlidmaatschap"
-          >Word lid</a
-        >
+        <a class="button primary rounded" href="/lid-worden">Word lid</a>
```

Comment:
Pls als je linkt naar een andere pagina op de site, gebruik dan `<router-link to="/pagina">tekst</router-link>`. Op die manier worden de components dynamisch omgewisseld ipv dat alle javascript opnieuw geladen wordt.
```suggestion
        <router-link class="button primary rounded" to="/lid-worden">Word lid</router-link>
```